helm/fasttrackml/values.yaml

```diff
@@ -0,0 +1,101 @@
+---
+replicaCount: 1
+
+image:
+  repository: gresearch/fasttrackml
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+data:
+  # common service settings.
+  FML_DEV_MODE: true
```

Comment:
We should add `FML_DATABASE_URI` as an environment var -- with examples for sqlite and postgres